CMakeLists.txt

```diff
@@ -180,6 +183,14 @@ endif ()
 # Enable unit tests.
 option(OPENASSETIO_ENABLE_TESTS "Create test targets" OFF)
 
+cmake_dependent_option(
+    OPENASSETIO_ENABLE_PYTHON_TEST_VENV
```

Comment:
[fix] Can we add this to the status dump output when you run `cmake -S`?

Reply:
Added. Now part of [[Build] Allow test execution without venv ](https://github.com/feltech/OpenAssetIO/commit/c66b14d9ac9e036690515e4e813c32cd6ce2f23b)